techmatters/flex-plugins
# Prevent hangup in conference if participant is on hold

## Description
This PR adds logic to prevent the counselor to hangup in a conference if there's at least one participant on hold. 

### Checklist
- [x] [Corresponding issue has been opened](https://tech-matters.atlassian.net/browse/CHI-1896)
- [n/a] Feature flags added
- [n/a] Tested for chat contacts

### Verification steps
- Start development server with `npm run start`.
- Start a voice contact and accept it with the above Flex instance.
- Add an external participant to the conference.
- Put any (or both) of the participants on hold.
- Confirm you can't leave the call unless participants are removed from hold. Also confirm an error notification is displayed to the counselor.
